db/migrate/20210214234012_add_denied_to_teachers.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeniedToTeachers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :teachers, :denied, :boolean
```

Comment:
Perhaps consider merging validated and denied into just a status (pending, approved, denied)

Reply:
Future story!